build_extra/rust/rust.gni

```diff
@@ -2,6 +2,10 @@ declare_args() {
   # Absolute path of rust build files.
   rust_build = "//build_extra/rust/"
 
+  # Wrapper executable for rustc invocations. This can be used for a caching
+  # utility, e.g. sccache.
+  rustc_wrapper = ""
```

Comment:
Add "Fixes #690" to your PR or the commit "build: add support for rustc_wrapper"

Reply:
done